K2Bridge/RewriteRules/RewriteSearchRule.cs

```diff
@@ -65,12 +67,23 @@ public async void ApplyRule(RewriteContext context)
                     context.HttpContext.Request.Path = $"/Query/SingleSearch/{GetIndexNameFromPath(context.HttpContext.Request.Path)}";
                 }
```

Comment:
L57-63 (and line 66) in the original file, can be removed ( I think its not used)

Reply:
Removed.